azurelinuxagent/common/event.py

```diff
@@ -775,8 +811,8 @@ def add_periodic(delta, name, op=WALAEventOperation.Unknown, is_success=True, du
                           message=message, log_event=log_event, force=force)
 
 
-def collect_events(reporter=__event_logger__):
-    return reporter.collect_events()
+def collect_events(enqueue_event, reporter=__event_logger__):
```

Comment:
suggestion: you may want to do a couple of renames here:

   collect_events doesn't collect events anymore, now it just applies a function to them... process_events?
   enqueue_event doesn't have to enqueue events, it is an arbitrary function that is applied to events... proceess_event_opeation?

Reply:
Done